src/FakeItEasy/Analysis/MustUseReturnValueAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace FakeItEasy.Analysis
+{
+    using System;
+
+    /// <summary>
+    /// Indicates that the return value of a method should not be ignored.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public sealed class MustUseReturnValueAttribute : Attribute

```

Comment:
There is still something that I do not understand, it seems. I've tried again, making `MustUseReturnValueAttribute` internal, and not only do things compile, but all your tests pass. I thought your original comment would preclude that.